server/text_generation_server/models/custom_modeling/flash_llama_modeling.py

```diff
@@ -299,6 +320,7 @@ def __init__(self, prefix, config, weights, index):
     def forward(self, hidden_states, adapter_data):
         if (
             SYSTEM == "rocm"
+            and False
```

Comment:
@drbh this was broken by the adapter PR

Reply:
whoops sorry about that @fxmarty! I've just opened a PR that should resolve https://github.com/huggingface/text-generation-inference/pull/2141

Reply:
Thank you!